Add FreeBSD support and forward-zones

Godwottery opened the pull request:
Added support for installing powedns-recursor on FreeBSD
Added support for managing forward-zones from Ansible

pieterlexis commented on tasks/configure.yml:
```diff
@@ -24,15 +24,15 @@
     src: recursor.conf.j2
     dest: "{{ pdns_rec_config_dir }}/{{ pdns_rec_config_file }}"
-    owner: "root"
+    owner: "{{ pdns_rec_user }}"
```
```suggestion
    owner: "{{ pdns_fileowner.user }}"
```

Godwottery replied:
Updated with the correct entity.

pieterlexis commented on tasks/configure.yml:
```diff
@@ -41,16 +41,16 @@
     dest: "{{ pdns_rec_config_lua }}"
     content: "{{ pdns_rec_config_lua_file_content }}"
-    owner: "root"
+    owner: "{{ pdns_rec_user }}"
```
```suggestion
    owner: "{{ pdns_fileowner.user }}"
```

Godwottery replied:
I have changed the hash to pdns_rec_file_owner. 
There are now two different entries, one for the owner of the configuration file, and one for the user that runs the service. 
Question from me: should these be the same in all cases, or would there ever be a case when they might be different? If they are always the same, we can merge the two variables.

pieterlexis replied:
They don't have to be the same. Users can have reasons to e.g. have it owned by `root:adm`

Godwottery replied:
Updated with the correct entity.

Godwottery commented:
> Can you add this to `defaults/main.yml`:
Done 

> Can you mention in the README that on FreeBSD, the repository options are not honored?
Added

> Could you add FreeBSD to the supported systems in `meta/galaxy.yml`?
Fixed in meta/main.yml, under the galaxy section. I assume that is what you meant. 

> I'm not a fan of using hashes for config options, perhaps changing `pdns_fileowner` to `pdns_rec_file_owner` and `pdns_rec_file_group` is cleaner.
See comment above

> And YAML lint is complaining:
Fixed

Godwottery commented:
Test failure: ```fatal: [centos-6]: FAILED! => {"changed": false, "msg": "Failure talking to yum: Cannot find a valid baseurl for repo: base"}```
Does not seem to be causes by this PR.

pieterlexis commented:
I've re-created your contribution in #64 as it diverged too much. Thanks for starting this!
